app/controllers/api/requests_controller.rb

```diff
@@ -21,8 +19,20 @@ def index
 
   private
 
+  def render_error_message(errors)
+    if errors.full_messages.one?
+      error_message = errors.full_messages.first
```

Comment:
Why not just call `.to_sentence` on `errors.full_messages`?

Reply:
bad knowledge from my side.... I will try it out!